scripts/wasmtestreport.py

```diff
@@ -254,6 +280,21 @@ def compile_and_run_native(source_file, timeout_sec=DEFAULT_TIMEOUT):
         if native_output.exists():
             native_output.unlink()
 
+        # Restore any executable dependencies that were swapped out
+        for dest_path in created_native_execs:
+            try:
+                if dest_path.exists():
+                    dest_path.unlink()
```

Comment:
Could we consider using  `dest_path.unlink(missing_ok=True)` to simplify the exists check instead?
And/or possibly narrow the exception grabbing to `FileNotFoundError` / `PermissionError` instead of a broader `Exception` which might swallow actual bugs?